viYarn/Episode/view/EpisodeTableViewController.swift

```diff
@@ -40,30 +40,40 @@ class EpisodeTableViewController: UITableViewController {
         title.textAlignment = .center
         title.textColor = .white
   
-        let btnDone = UIBarButtonItem(title: "Done", style: .done, target: self, action: #selector(closeView))
+        let btnDone = UIBarButtonItem(title: "Done", style: .done, target: self, action: #selector(onClickDone))
         btnDone.tintColor = .white
         navigationItem.rightBarButtonItem = btnDone
     }
     
-    @objc func closeView(){
-        
-    }
     
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return arr.count
     }
     
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell:EpisodeTableViewCell = tableView.dequeueReusableCell(withIdentifier: "cell") as! EpisodeTableViewCell
-        cell.setData(data: arr[indexPath.row])
-        cell.backgroundColor = colorBackground
-        cell.selectionStyle = .none
-        return cell
+        if let cell:EpisodeTableViewCell = tableView.dequeueReusableCell(withIdentifier: EpisodeTableViewCell.CELL_KEY) as? EpisodeTableViewCell
+        {
+            cell.delegate = self
+            cell.setData(data: arr[indexPath.row])
+            cell.backgroundColor = colorBackground
```

Comment:
p0: Create a func configured(with episode: Episode) -> CellType, in which we will configure cell UI.